js/src/edit-block/helpers/getSettingsComponent.js

```diff
@@ -39,9 +39,8 @@ const getSettingsComponent = ( settingType ) => {
 	}
 
 	const componentName = snakeCaseToPascalCase( settingType );
-
-	const filteredComponents = applyFilters( 'genesisCustomBlocks.settingsComponents', settingsComponents );
-	return filteredComponents[ componentName ] ? filteredComponents[ componentName ] : null; /* eslint-disable-line import/namespace */
+	const component = settingsComponents[ componentName ] ? settingsComponents[ componentName ] : null; /* eslint-disable-line import/namespace */
+	return applyFilters( 'genesisCustomBlocks.getSettingsComponent', component, componentName );
```

Comment:
@kienstra Can you help me understand why this filter is here?

Reply:
Sure, it's so the pro plugin can add settings components for pro fields in https://github.com/studiopress/genesis-custom-blocks-pro/pull/16/files#diff-bceaff574c921b662c2f63f1c23f001186651eb712c7a46a362c2cdea388b78bR12

Reply:
For example, the Post field's 'Post Type' settings component:
![pro-isdf](https://user-images.githubusercontent.com/4063887/102383073-c0f42b80-3f90-11eb-8236-23f9769ec7bc.png)

Reply:
Cool thanks!